### PR TITLE
Build project in Github actions

### PR DESCRIPTION
**Pull request by thhous-msft**

### CODE REVIEW

**Review by nibanks — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: etl2pcapng Build
+
+on:
+  push:
+    branches:
+    - master
```
Comment (nibanks) [inline .github/workflows/build.yml:6]: We should rename this to `main`.

Comment (nibanks) [reply .github/workflows/build.yml:6]: this == the primary branch, not just this string.

Comment (thhous-msft) [reply .github/workflows/build.yml:6]: Agreed. Do you remember the steps to do this?

Comment (nibanks) [reply .github/workflows/build.yml:6]: Merge this PR and then I can help with this.

**Pull request merged**